.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
 *__pycache__
+.env
```

Comment:
Just one small thing, I see people provide a `.env.example` in their repositories, which I believe tells us the keys which one would need to add there, something like this:
```
ACCESS_TOKEN_EXPIRE_MINUTES=<your_token_here>
```

Reply:
Added!